### ISSUE TITLE
[Question] - Where do I look for AsyncStorage logs?

### ISSUE THREAD

**Issue by tolu360** [infinitered/reactotron#379]
First off, thank you for this awesome product. Extremely neat and useful.

I am just getting started with `Reactotron`, wiring it up in my `react-native` project was a breeze. I hooked up the `asyncStorage` plugin but I clearly don't know where to look in `Reactotron` UI for the logs. My set up looks like:

```javascript
import Reactotron, { asyncStorage } from 'reactotron-react-native';
import { reactotronRedux } from 'reactotron-redux';

Reactotron
	.configure()
	.use(reactotronRedux())
	.use(asyncStorage());

if (__DEV__) Reactotron.connect();
```

Please, am I missing something? My version of `Reactotron` says 1.6. Thank you

**Comment by skellock** [comment]
Heyo... you'll need the latest build (1.9) in your codebase as well as the app to see it.

**Comment by skellock** [comment]
oh and ty for the kind words.  =)

**Comment by tolu360** [comment]
You are very welcome @skellock. Thank you for one of the fastest responses have ever got! Now downloading the latest from `releases`. I guess the `getting started` instructions [here](https://github.com/infinitered/reactotron/blob/master/docs/quick-start-react-native.md) are now a bit dated. Cheers

**Comment by skellock** [comment]
Oops!  Thank you for telling me.  Fixed with #380.

**Comment by gamingumar** [comment]
My AsyncStorage logs don't show up, I am using ```"reactotron-react-native": "^3.6.4",```
```
Reactotron.configure()
    .use(asyncStorage())
    .connect();

Reactotron.useReactNative({
    asyncStorage: {
      ignore: ['secret']
    },
    networking: {
      ignoreUrls: /symbolicate/
    },
    editor: true,
    errors: { veto: stackFrame => false },
    overlay: true
  })
```
